pkg/pipelines/service.go

```diff
@@ -38,7 +41,7 @@ func AddService(o *AddServiceOptions, appFs afero.Fs) error {
 	if err != nil {
 		return err
 	}
-	files, err := serviceResources(m, appFs, o)
+	files, otherResources, err := serviceResources(m, appFs, o)
```

Comment:
Segregating the resources in the same map would help us here as well

Reply:
As discussed, we'll keep the two resource lists separate.